js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
```

Comment:
All `kwargs` are sent up from the server, as usual.

It's not being used right now, but here's an old Seasonal that used it:

https://github.com/Zarel/Pokemon-Showdown/blob/a8ba012e1e7f9ae14010473a16b453ee6be65e51/config/formats.js#L665-L675